clikit/src/main/java/org/adamalang/clikit/Tool.java

```diff
@@ -0,0 +1,90 @@
+package org.adamalang.clikit;
+
+import org.adamalang.clikit.codegen.*;
+import org.adamalang.clikit.exceptions.XMLFormatException;
+import org.adamalang.clikit.model.ArgumentDefinition;
+import org.adamalang.clikit.model.Command;
+import org.adamalang.clikit.model.Common;
+import org.adamalang.clikit.model.Group;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.SAXParserFactory;
+import javax.xml.parsers.SAXParser;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Tool {
+    /** Tool to parse XML and create files according to the data **/
```

Comment:
move to prior the "public class" as that will associate the comment with the class